### ISSUE TITLE
for android ios like rounded checkbox needed

### ISSUE THREAD

**Issue by theshahzaib5** [react-native-checkbox/react-native-checkbox#108]
please add support for android to add IOS like rounded checkbox, or please add support to add a custom icon support.

**Comment by nicholaslee119** [comment]
Hi, @theshahzaib5  Thanks for your report.
The react-native-checkbox for android is implemented with the native checkbox in android. As I know, there is no way to customize the rounded style of the checkbox in native way. If there are, please let me know.

**Comment by theshahzaib5** [comment]
Hello, @nicholaslee119 

I think you can use a custom View or custom TouchableOpacity, to achieve this, or you can add props for custom checked, unchecked icons.

**Comment by nicholaslee119** [comment]
@theshahzaib5 
Yes, Indeed, you could implement by yourself like that. The responsibility of the repo is just to provide an RN way to access the native checkbox.
